static/compiler.js

```diff
@@ -504,13 +504,12 @@ define(function (require) {
     };
 
     Compiler.prototype.onCompilerSetDecorations = function (id, lineNums) {
-        if (id === this.id) {
+        if (id == this.id) {
```

Comment:
That's super weird! Leave it as is. I'll have a think about fixing it in a more centralised way. Thanks!